lms/templates/main.html

```diff
@@ -61,6 +61,21 @@
 
   <%static:css group='style-vendor'/>
   <%static:css group='${self.attr.main_css}'/>
+  ## We could do <%static:css group='style-main'/>, but that's only useful
+  ## if the group contains multiple files, and the 'style-main' group doesn't.
+  ## Instead, we'll construct this <link> element manually, to improve clarity.
+  ## When nothing in the system is referencing the 'style-main' group, it can
+  ## be removed from the environment file.
+  <%
+  application_css_path = "css/lms-main{rtl}.css".format(
+    rtl="-rtl" if get_language_bidi() else "",
+  )
+  application_css_path_pattern = "/static/edx-pattern-library/pattern-library/css/edx-pattern-library-{rtl}.min.css".format(
+    rtl="rtl" if get_language_bidi() else "ltr",
+  )
+  %>
+  <link rel="stylesheet" href="${static.url(application_css_path)}" type="text/css" media="all" />
+  <link rel="stylesheet" href="${application_css_path_pattern}" type="text/css" media="all" />
 
```

Comment:
@douglashall , when I remove these lines then edx pattern library does not include. I have changed the lms-main into lms-main-v2 as according to your branch.